data-plane/core/src/test/java/dev/knative/eventing/kafka/broker/core/EgressWrapperTest.java

```diff
@@ -18,7 +18,7 @@
 
```

Comment:
Can you add a couple of tests since we added some fields, the same applies to `ResourceWrapperTest`?